README.md

```diff
@@ -0,0 +1,42 @@
+# Quai CPU Miner
```

Comment:
https://github.com/dominant-strategies/quai-cpu-miner/pull/11 Also creates a readme. Might have to merge sequentially.